dotnet-desktop-guide/net/winforms/data/snippets/how-to-synchronize-multiple-controls/csharp/Form1.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace binding_control_example
+{
+    public partial class Form1 : Form
+    {
+        public Form1()
+        {
+            //<bind_controls_using_BindingSource>
```

Comment:
This should be moved up so that the { } are totally visible. Currently the ending } is visible but the opening { isn't